.github/workflows/ubuntu.yml

```diff
@@ -92,6 +92,10 @@ jobs:
             otp-version: '24'
             rebar3-version: '3.15'
             os: 'ubuntu-20.04'
+          - elixir-version: 'v1.11.0'
```

Comment:
Added because an issue was found live for this specific combo (and also because `'version-type': 'strict'` was not being tested in the GitHub workflows).